0_intro/exercises/vec_strlen/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 pub fn vec_strlen(vec: Vec<&str>) -> Vec<usize> {
-    todo!()
+    let mut res = Vec::new();
+    vec.iter().for_each(|s| res.push(s.len()));
```

Comment:
Very good! If you want an extra challenge, you can also do it in one line using `map` and `collect`. Here's the documentation for this method:

https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect